src/components/VendorDashboard/VendorDashboard.js

```diff
@@ -49,22 +49,21 @@ class VendorDashboard extends Component {
     const { isAuthenticated, user, services } = this.props;
 
     if (isAuthenticated && user.isVendor === true) {
-      const length = services.length;
       let data = [];
-      // Adding elements to the data array
-      if (length) {
-        data = services.map((obj,index) => {
-          const {name,detail} = obj
-          const {contact,location} =detail;
+        data  = services.map((data,index)=>{
+          console.log(data,index);
```

Comment:
Remove this. In your editor user search to find and remove unnecessary console.log

Reply:
Try switching the tabs in the 1,2,3,4 they give a message an error occurred.
To reproduce.
Enter the business name.
Click on done.
Click on tab 1

Reply:
Made the changes , corrected the bug